### ISSUE TITLE
ectool and windows support

### ISSUE THREAD

**Issue by leopoldhub** [TamtamHero/fw-fanctrl#30]
Hi,

In the PR #29 , we replaced Most of the Linux dependencies (replacing the file paths with ectool) except for the sockets.

It should now be possible to adapt the script for windows users.

The ectool can be built for win x64 by reproducing the goal in the `.gitlab-ci.yml` file (after installing VisualStudio Community 2022 with the C++ and ClangCL package (the installation path may vary slightly))
I have successfully built it in a VM, however since I do not have a native windows installation, I could not fully test if the ectool can properly interacts with the computer hardware.

If you have one, could you please test it on your end, and if everything is ok add the exe to the repo bin folder so I can start the adaptation?

**Comment by TamtamHero** [comment]
Indeed, that would be nice for Windows users but I don't have a Windows install either

**Comment by leopoldhub** [comment]
I will try to setup one without messing with my existing install then
